dvc/repo/__init__.py

```diff
@@ -463,6 +472,13 @@ def brancher(self, *args, **kwargs):
 
         return brancher(self, *args, **kwargs)
 
+    @contextmanager
+    def switch(self, rev: str) -> Iterator[str]:
+        from dvc.repo.brancher import switch
+
+        with switch(self, rev) as rev:
+            yield rev
+
```

Comment:
Nitpicky suggestion, so no strong opinion, doing similar to brancher above.
```suggestion
    @contextmanager
    def switch(self, rev: str) -> Iterator[str]:
        from dvc.repo.brancher import switch

        return switch(rev)
```